pkg/cmd/codespace/delete.go

```diff
@@ -80,7 +80,7 @@ func (a *App) Delete(ctx context.Context, opts deleteOptions) (err error) {
 	nameFilter := opts.codespaceName
 	if nameFilter == "" {
 		a.StartProgressIndicatorWithLabel("Fetching codespaces")
-		codespaces, err = a.apiClient.ListCodespaces(ctx, -1, opts.orgName, opts.userName)
+		codespaces, err = a.apiClient.ListCodespaces(ctx, api.ListCodespacesOptions{OrgName: opts.orgName, UserName: opts.userName})
```

Comment:
At first read, I was nervous this would translate to `limit=0` in the request, but there's a check that the limit is greater than 0, so this should work. I'm leaving this comment in case anyone else has a similar thought.

https://github.com/cli/cli/pull/5854/files#diff-03f3956c875564cb8b57e93c52220bbc278ed4c250bd83db38040d016651a585R287